test/test_utils.py

```diff
@@ -689,6 +689,14 @@ def test_hub_dir(self):
             torch.hub.set_dir(dirname)
             self.assertEqual(torch.hub.get_dir(), dirname)
 
+    def test_hub_dir_expand(self):
+        with tempfile.TemporaryDirectory('cur_dir') as dirname:
```

Comment:
@loodvn Thanks for the PR, and for your patience. The PR looks good, but I'm wondering why we need a tempdir here? I think it comes from the previous test above, but we don't seem to use it in this test.